package.json

```diff
@@ -277,7 +277,8 @@
       "atom-beautify:beautify-language-xml",
       "atom-beautify:beautify-language-xtemplate",
       "atom-beautify:beautify-language-yaml",
-      "atom-beautify:beautify-language-terraform"
+      "atom-beautify:beautify-language-terraform",
+      "atom-beautify:beautify-language-edge"
```

Comment:
These look like they are in alphabetical order, meaning edge should be further up.

Reply:
Did a proper sorting using vims sorting feature.